src/aws-cpp-sdk-core/source/smithy/client/AwsSmithyClientBase.cpp

```diff
@@ -102,7 +103,17 @@ void AwsSmithyClientBase::baseCopyAssign(const AwsSmithyClientBase& other,
   m_serviceUserAgentName = other.m_serviceUserAgentName;
   m_httpClient = std::move(httpClient);
   m_errorMarshaller = std::move(errorMarshaller);
-  m_interceptors = Aws::Vector<std::shared_ptr<interceptor::Interceptor>>{Aws::MakeShared<ChecksumInterceptor>("AwsSmithyClientBase")};
+  
+  // Create modified config for chunking interceptor
+  Aws::Client::ClientConfiguration chunkingConfig(*m_clientConfig);
```

Comment:
same comment as the other class, why create a temporary object, why not just inline the creation of the interceptor